### ISSUE TITLE
Regression in 1.3.1 with parameters that start with `_` no longer valid

### ISSUE THREAD

**Issue by clinejj** [ruby/rbs#722]
After updating from RBS 1.2 to 1.3.1, parameters that start with an underscore are no longer valid. Example from [ruby/gem_rbs_collection](https://github.com/ruby/gem_rbs_collection):

https://github.com/ruby/gem_rbs_collection/blob/main/gems/actionpack/6.0/actionpack-generated.rbs#L5750-L5753
```rb
    # Aliasing these off because AD::Http::Cache::Response defines them.
    alias _cache_control cache_control

    alias _cache_control= cache_control=
```

yields this error when running `steep_check`:

```sh
#<RBS::Parser::SyntaxError: parse error on value: #<RBS::Parser::LocatedValue:0x00007f8bd14dd5f0 @location=#<RBS::Location:5240 @buffer=/Users/john/Documents/GitHub/repo/gem_rbs/gems/actionpack/6.0/actionpack-generated.rbs, @pos=210321...210335, source='_cache_control', start_line=5751, start_column=10>, @value="_cache_control"> (tUNDERSCOREIDENT)>
```

It looks like https://github.com/ruby/rbs/pull/721 was an attempt to address this bug but didn't totally fix it.

Also filed https://github.com/ruby/gem_rbs_collection/issues/44 in case it's an issue with the generated RBS files in the repo.

**Comment by pocke** [comment]
Thanks for reporting the issue. I've opened a PR to fix this problem. #723

**Comment by soutaro** [comment]
I'm sorry for the regression. Released 1.3.2, and hope the version works for you.

**Comment by clinejj** [comment]
Just verified, everything looks great. Thanks for the quick fixes @pocke and @soutaro!
